sdk/appconfig/azappconfig/policy_hmac_auth.go

```diff
@@ -69,8 +69,8 @@ func (policy *hmacAuthenticationPolicy) Do(request *policy.Request) (*http.Respo
 	}
 
 	req.Header.Set("x-ms-content-sha256", contentHash)
-	req.Header.Set("x-ms-date", timestamp)
-	req.Header.Set("Authorization", "HMAC-SHA256 Credential="+id+", SignedHeaders=x-ms-date;host;x-ms-content-sha256, Signature="+signature)
+	req.Header.Set("Date", timestamp)
+	req.Header.Set("Authorization", "HMAC-SHA256 Credential="+id+", SignedHeaders=date;host;x-ms-content-sha256, Signature="+signature)
```

Comment:
Does `Date` need to be capitalized in the header? Otherwise, LGTM

Reply:
Good point, but I'd leave it capitalized for 3 reasons:
1. This is what .NET SDK does
2. This is what is confirmed to work
3. In RFCs, such as RFC 2616, "Date" and "Authorization" are capitalized.

Reply:
What about in the Authorization header ie `SignedHeaders=date;...` vs `SignedHeaders=Date`

Reply:
I would not change it, this is how .NET and also Java put it, and it works.

Reply:
BTW, do you know if Go SDK does lowercase headers? I know we do it in C++ SDK.